lib/src/cli.rs

```diff
@@ -234,6 +239,10 @@ pub(crate) enum ContainerImageOpts {
         /// the new manifest.
         #[clap(long)]
         check: Option<Utf8PathBuf>,
```

Comment:
It looks like this Option is not respected by clap and always asked on the command line:

```
$ ostree container image pull --check check repo ostree-image-signed:registry:a.a/a:a
error: Opening directory at 'repo': No such file or directory (os error 2)

$ ostree container image pull --check repo ostree-image-signed:registry:a.a/a:a
error: the following required arguments were not provided:
  <IMGREF>

Usage: ostree-container container image pull --check <CHECK> <REPO> <IMGREF>

For more information, try '--help'.
```

Reply:
Hum, OK, it's a little bit more complex than that for the pull --check option as it optionally writes to a file.